python_practice/calculator/simple_calc.py

```diff
@@ -96,28 +82,29 @@ def main():
     )
 
     if operator_input == '+':
-        print(add(
+        result = add(
             input_x,
             input_y,
-        ))
+        )
     elif operator_input == '-':
-        print(subtract(
+        result = subtract(
             input_x,
             input_y,
-        ))
+        )
     elif operator_input == '/':
-        print(divide(
+        result = divide(
             input_x,
             input_y,
-        ))
+        )
     elif operator_input == '*':
-        print(multiply(
+        result = multiply(
             input_x,
             input_y
-        ))
+        )
     else:
         print('enter a valid operator')
```

Comment:
This should also be a result, and should probably be 'did not enter a valid operator'. Otherwise you might get double print